Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1763,6 +1905,25 @@ private void OnDisable()
         /// <summary>
         /// Debug helper method that can be hooked up to actions when using <see cref="UnityEngine.InputSystem.PlayerNotifications.InvokeUnityEvents"/>.
         /// </summary>
+        /// <param name="context">Information about what triggered the action.</param>
+        /// <remarks>
+        /// <example>
+        /// <code>
+        /// using UnityEngine;
+        /// using UnityEngine.InputSystem;
+        /// // Component to sit next to PlayerInput.
+        /// [RequireComponent(typeof(PlayerInput))]
+        /// public class MyPlayerLogic : MonoBehaviour
```

Comment:
Enabling and disabling this component multiple times would add one new each handler on every OnEnable? Is this intended

Reply:
Nope I'll add an OnDisable function too.